package.json

```diff
@@ -19,12 +19,16 @@
         "crowdin:upload": "tsx resources/js/tools/crowdin-upload.ts"
     },
     "dependencies": {
+        "@bbob/plugin-helper": "^4.2.0",
+        "@bbob/preset-react": "^4.2.0",
+        "@bbob/react": "^4.2.0",
         "@floating-ui/core": "^1.6.8",
         "@floating-ui/dom": "^1.5.1",
         "@hookform/resolvers": "^3.9.0",
         "@inertiajs/core": "^1.2.0",
         "@radix-ui/react-alert-dialog": "^1.1.4",
         "@radix-ui/react-checkbox": "^1.1.1",
+        "@radix-ui/react-collapsible": "^1.1.2",
```

Comment:
This powers a base component for the "Spoiler" button, but can also be used for other collapses, such as the activity cards on the new activity page.